### PR TITLE
Very small edits to context diagram

### PR DESCRIPTION
**Pull request by lwjohnst86**

### CODE REVIEW

**Comment by lwjohnst86** [comment]
If someone could review this, that'd be awesome!

File: design/images/component-api-and-auth.puml [hunk 1]
```diff
@@ -1,42 +1,40 @@
...
+!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Component.puml
 
-' C4 Component diagram for The Seedcase product - API
-
-Person(direct_access, "Direct connection")
-Person(admin, "Admin user")
 Container(frontend, "Front end")
-Container(backend, "Back end")
-Container(uia, "User ID Authentication")
+Container(bee, "Back end environment")
```
Comment (rqding) [inline design/images/component-api-and-auth.puml:6]: I think Backend is one word

File: design/images/component-api-and-auth.puml [hunk 2]
```diff
@@ -1,42 +1,40 @@
...
-!includeurl https://raw.githubusercontent.com/RicardoNiepel/C4-PlantUML/master/C4_Component.puml
+@startuml component-api-and-auth
+' C4 Component diagram for the API and Authentication modules
+!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Component.puml
 
-' C4 Component diagram for The Seedcase product - API
-
-Person(direct_access, "Direct connection")
-Person(admin, "Admin user")
 Container(frontend, "Front end")
```
Comment (rqding) [inline design/images/component-api-and-auth.puml:5]: Same here as the term frontend

Comment (signekb) [reply design/images/component-api-and-auth.puml:5]: If that's the case, it should be changed in the text as well

File: design/images/context.puml [hunk 3]
```diff
@@ -3,13 +3,13 @@
...
 Person(user_data_request, "Data discover and request")
-Person(user_admin, "Admin")
+Person(user_admin, "Admin", "Data owners, data managers, and other ")
 
 Person_Ext(user_results, "Explore findings and projects")
 
 Lay_Right(user_admin, user_results)
 
-System(ui, "Seedcase System for Data Resource")
+System(ui, "Seedcase Instance for a Data Resource")
```
Comment (signekb) [inline design/images/context.puml:12]: "Instance of a Seedcase Data Resource"?

File: design/images/context.puml [hunk 4]
```diff
@@ -3,13 +3,13 @@
 
 Person(user_data_input, "Data input")
 Person(user_data_request, "Data discover and request")
-Person(user_admin, "Admin")
+Person(user_admin, "Admin", "Data owners, data managers, and other ")
```
Comment (signekb) [inline design/images/context.puml:6]: If data "owners" and "managers" are added here, should they be described in the text as well?

File: design/images/component-api-and-auth.puml (hunk 1 above)
Comment (signekb) [reply design/images/component-api-and-auth.puml:6]: If that's the case, it should be changed in the text as well

File: design/images/context.puml (hunk 4 above)
Comment (lwjohnst86) [reply design/images/context.puml:6]: Good idea, I've synch'd the text

File: design/images/context.puml (hunk 3 above)
Comment (lwjohnst86) [reply design/images/context.puml:12]: Hmmm, thinking on it, I'm not sure either sounds right to me. I've modified it, maybe this is better. We'll have more reviews of this over time.

**Pull request merged**